services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -28,7 +28,16 @@ class ProjectFileService(
   ): Future[Either[VersioningException, UpdateFiledResponse]] =
     projectService
       .getUserProjectById(projectId, userId)
-      .flatMap(project => projectVersioning.updateFile(project, projectFile, version))
+      .flatMap(
```

Comment:
```suggestion
      .flatMap {
```
Is more convenient to use curly braces in case of multiline statements. Besides, you already use curly braces on line 33, so it would be more consistent.